### PR TITLE
Create LIT6947HymnJesus

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a LIT ID for LIT6947HymnJesus according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2505.

### CODE REVIEW

File: new/LIT6947HymnJesus.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <change who="CH" when="2024-03-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div xml:lang="gez" type="edition">
+                <note>The following incipit is taken from the first stanza from <ref type="mss" corresp="EMIP02368"/>.</note>
+                <div type="textpart" subtype="incipit">
+                    <ab>
+                        ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡
```
Comment (thea-m) [inline new/LIT6947HymnJesus.xml:63]: It is possible to encode verses with `<l="1">ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡</l>`

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

File: new/LIT6947HymnJesus.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6947HymnJesus.xml:63]: XML does not allow `<ab>` to be followed by `<l>`. How can I resolve?
```
<div type="textpart" subtype="incipit">
                    <ab>
                    <l="1">ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡</l>    
                    <l="2">እስራኤል፡ ወይሁዳ፡ እንተ፡ ኃለፉ፡ በውሥጣ፡</l>
                    <l="3">በክቡር፡ ደሙ፡ ለነፍስየ፡ ዘተሣየጣ፡</l>
                    <l="4">ካዕበኒ፡ በሰበነ፡ ርእሱ፡ ዘሦጣ፡</l>
                    <l="5">ለማርያም፡ ንጽሕት፡ አፍአ፡ ወውሥጣ። </l> 
                    </ab>
                </div>
```

Comment (eu-genia) [reply new/LIT6947HymnJesus.xml:63]: by not using `ab `when you have` l`

Comment (CarstenHoffmannMarburg) [reply new/LIT6947HymnJesus.xml:63]: But the first line with `<l>` is still marked with a red line.

```
<div xml:lang="gez" type="edition">
                <note>The following incipit is taken from the first stanza from <ref type="mss" corresp="EMIP02368"/>.</note>
                <div type="textpart" subtype="incipit">
                    <l="1">ይትባረክ፡ እግዚአብሔር፡ ለባሕረ፡ ኤርትራ፡ ዘሠጠጣ፡</l>    
                    <l="2">እስራኤል፡ ወይሁዳ፡ እንተ፡ ኃለፉ፡ በውሥጣ፡</l>
                    <l="3">በክቡር፡ ደሙ፡ ለነፍስየ፡ ዘተሣየጣ፡</l>
                    <l="4">ካዕበኒ፡ በሰበነ፡ ርእሱ፡ ዘሦጣ፡</l>
                    <l="5">ለማርያም፡ ንጽሕት፡ አፍአ፡ ወውሥጣ። </l> 
                </div>
            </div>
```

Comment (thea-m) [reply new/LIT6947HymnJesus.xml:63]: Sorry, my typo. I meant  `<l n="1">`

**Pull request merged**